average.js

```diff
@@ -1,6 +1,8 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  
+  //Used Filter Function to clear out any Garbage inputs.
+   numbers=numbers.filter(Boolean);
+   return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
 }
 
 module.exports = {average};
```

Comment:
Try a new test: The tests in `test/average.test.js` express the functionality of the `average` function.
Consider this situation: In IoT applications, analog sensors will report occasional outliers. 'Outliers' are values so small or so large, they aren't expected to happen. We need new functionality to ignore these outliers.
First add a unit-test in `test/average.test.js` to supply outliers and expect them to be ignored. Name the test appropriately. Then implement it in the code.
You'll need thresholds to recognize outliers: Think of sensing battery-temperature values and determine reasonable values.

Reply:
Cool , I like this challenging idea and will soon update you about the progress in it. 
But is it also mean my application will be accepted by your organization?

Reply:
Hey what can be the thresholds or criteria to recognize outliers?